### DISCUSSION TITLE
멀티플레이는 유니티-> 제페토 큐알 스캔으로 테스트 불가능한가요?

### DISCUSSION THREAD

**Discussion by Hello98752** [naverz/zepeto-studio-kor#1126] [5. Q&A]
안녕하세요

샘플 중 world-movingblock-sample-main과 zepeto-multiplay-example-main을 다운받았습니다.
그리고 따로 터치한 것 없이 큐알코드로 제페토 내에서 시범 플레이를 해보려했습니다.
그런데 제대로 작동하지않아, 혹시 멀티플레이는 테스트가 불가능한건지 궁금합니다.

![image](https://user-images.githubusercontent.com/110609361/182845583-8bd5c591-1ba0-4a6c-a5f7-8733cc2dda5b.png)
이 씬은
![image](https://user-images.githubusercontent.com/110609361/182845858-73b47bf4-9b21-4eb9-b35f-86dfcd4624c2.png)
이렇게 보이고, (캐릭터가 생성되지 않음)

![image](https://user-images.githubusercontent.com/110609361/182845910-75f1e918-f8a8-480b-9651-4fc1b328ae63.png)
이 씬은
![image](https://user-images.githubusercontent.com/110609361/182845945-3ebe6d2f-6e3c-4939-a7d5-52bb37929cd1.png)

이렇게 보이는데요,
막상 출시해서 멀티플레이로 플레이하면 정상적으로 보이는건가요?
아니면 샘플에 문제가 있는 것인가요? 궁금합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이 월드는 상단의 멀티플레이 서버를 켠 후 테스트를 하셔야 합니다.
아래 가이드를 참고하여 주세요~
https://studio.zepeto.me/guides/multiplay
